test/api/controllers/authController.test.js

```diff
@@ -511,6 +511,16 @@ describe('Test the auth controller', () => {
         UnauthorizedError,
         {id: 'security.rights.unauthorized'});
     });
+
+    it('should not expires the token if this is an API Key', async () => {
+      Object.defineProperty(request.context.token, 'type', { get: () => 'apiKey'});
+      const response = await authController.logout(request);
+
+      should(kuzzle.ask)
+        .not.be.calledWith('core:security:token:delete', request.context.token);
+      
+      should(response.responseObject).be.instanceof(Object);
```

Comment:
What is this for? 😄

Reply:
Leftover from when I deleted unused test cases after changing the way we detect API Keys